client/config.go

```diff
@@ -66,8 +68,7 @@ func runConfigCmd(cmd *cobra.Command, args []string) error {
 		Home:      gaiaCLIHome,
 		ChainID:   chainID,
 		TrustNode: trustNode,
-		Encoding:  encoding,
-		Output:    output,
+		Output:    "text",
```

Comment:
let's please create and use a `newCliConfig(...)` function for here (avoid missing structs in the future)

Reply:
This whole functionality is currently broken. I'll open an issue for that and fix it in another PR if thats OK.